### ISSUE TITLE
Need official process for handling HS3 and HS4 devices

### ISSUE THREAD

**Issue by rmasonjr** [HomeSeer/Plugin-SDK#183]
I opened a ticket at HST, but would like to post here in addition:

@HST
We need some clear guidelines on how to go about handling plugin devices with customers who have old HS3 devices.

For example, if you have a plugin that has a parent device with 'x' child devices, what should the proper process be to convert those old HS3 devices to the new HS4 devices? Or, should we leave them alone? If we convert, how do we maintain the device reference Ids as not to break events? Will there be any Alexa/Google Home implications?

In short, it would be good to have something official from the HST team on how to handle HS3 to HS4 devices.
Thanks

**Comment by alexbk66** [comment]
Even better if PluginSdk included a single function for that
